store/sqlstore/post_store.go

```diff
@@ -1236,6 +1237,28 @@ func (s *SqlPostStore) search(teamId string, userId string, params *model.Search
 		searchClause := fmt.Sprintf("AND to_tsvector('english', %s) @@  to_tsquery('english', :Terms)", searchType)
 		searchQuery = strings.Replace(searchQuery, "SEARCH_CLAUSE", searchClause, 1)
 	} else if s.DriverName() == model.DATABASE_DRIVER_MYSQL {
+		if searchType == "Message" {
+			stopWords, err := findStopWordsMysql(s)
+			if err != nil {
+				return nil, model.NewAppError("SqlPostStore.search", "store.sql_post.search.app_error", nil, err.Error(), http.StatusInternalServerError)
+			}
+
+			for i := 0; i < len(stopWords); i++ {
```

Comment:
Given that you'd have a slice of strings you don't need this

Reply:
Not sure about this, maybe I don't understand the point :sweat_smile: